knowledge_graph/make_graph.py

```diff
@@ -255,73 +167,6 @@ def get_test_ontology(G, valid_test_ont, not_test_ont):
         remove_non_test_nodes(G, node_b, valid_test_ont, not_test_ont)
 
 
-def get_valid_test_ont():
-    return {
-        "test ontology",
-        "personal value",
-        "achievement",
-        "benevolence",
-        "benevolence caring",
-        "benevolence dependability",
-        "conformity",
-        "conformity interpersonal",
-        "conformity rules",
-        "face",
-        "hedonism",
-        "humility",
-        "power",
-        "power dominance",
-        "power resources",
-        "security",
-        "security personal",
-        "security societal",
-        "self-direction",
-        "self-direction autonomy of action",
-        "self-direction autonomy of thought",
-        "stimulation",
-        "tradition",
-        "universalism",
-        "universalism concern",
-        "universalism nature",
-        "universalism tolerance",
-    }
-
-
-def get_non_test_ont():
-    return {
-        "value uncategorized (to do)",
-        "risk solution",
-        "adaptation",
-        "geoengineering",
-        "indirect adaptation",
-        "indirect geoengineering",
-        "indirect mitigration",
-        "carbon pricing",
-        "carbon tax",
-        "emissions trading",
-        "mitigation",
-        "solution to indirect adaptation barrier",
-        "solution to indirect mitigation barrier",
-        "solution uncategorized (to do)",
-    }
-
-
-# Read the JSON file back
-def read_json_file(filename):
```

Comment:
we wanted to keep the function read_json_file so that other versions of the knowledge graph could be loaded in without needing to repeat the whole pipeline. This should be moved to utils and not deleted.

Reply:
It wasn't deleted, just moved. See line 103 of knowledge_graph/ontology_processing_utils.py

Reply:
that is for the saving function. Not for the reading function, right? 
`def read_json_file(filename):` I don't see in ontology_processing_utils.py

Reply:
Ah, my bad. Yeah, I think I removed this because it wasn't being used anywhere else in the codebase, and it's just an alias for `json.load(f)`. I can put it back in utils if you want to keep it, but I think it's clearer to just call `json.load`. It's basically self-documenting code. If you feel invoking `read_json_file` adds clarity let's go for it, but my feeling is that stacking layers of function definitions on top of self-explanatory operations just makes the code more confusing rather than less. I definitely don't want to take away convenience functions you find useful, adding it to utils is no big deal.

Reply:
Oh ok. If that’s the case then no need to have it.